kinto_attachment/utils.py

```diff
@@ -87,8 +87,10 @@ def patch_record(record, request):
     # Simulate update of fields.
     request.validated = record
     request.body = json.dumps(record).encode('utf-8')
+    request.bound_data['attachment_auto'] = True
```

Comment:
Why do you want to change the bound_data rather than another attribute on the request itself?